component/src/main/java/com/celements/photo/container/imageurl/helpers/AreaWithLimitsAndDefault.java

```diff
@@ -0,0 +1,38 @@
+package com.celements.photo.container.imageurl.helpers;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.celements.photo.container.ImageUrl;
+import com.google.common.base.Optional;
+
+public class AreaWithLimitsAndDefault {
```

Comment:
can I understand this to be a "helper" class for the predicates/comparators? than make it package private